src/user.c

```diff
@@ -181,6 +181,7 @@ Nick *AddNick(char *nick, char *ident, char *host, char *uid, char *hiddenhost,
     new_nick->ignoretime = 0;
     new_nick->loginattempts = 0;
     new_nick->lasttry = 0;
+    new_nick->svid[0] = '\0';
```

Comment:
Not needed. The entire nick structure is zero-ed with the memset above.

Reply:
Removed